xrmocap/transform/keypoints3d/optim/fourdag_base_optimizer.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+from xrmocap.utils.fourdag_utils import skel_info
+from xrmocap.ops.triangulation.builder import (
+    BaseTriangulator, build_triangulator,
+)
+from typing import Union
+
+
+class FourDAGBaseOptimizer():
+
+    def __init__(self,
+                 triangulator: Union[None, dict, BaseTriangulator] = None,
+                 kps_convention='fourdag_19',
+                 min_triangulate_cnt: int = 15,
+                 triangulate_thresh: float = 0.05,
+                 logger=None):
+
+        if isinstance(triangulator, dict):
+            self.triangulator = build_triangulator(triangulator)
+        else:
+            self.triangulator = triangulator
+
+        self.kps_convention = kps_convention
+        self.min_triangulate_cnt = min_triangulate_cnt
+        self.triangulate_thresh = triangulate_thresh
+
+        self.projs = None
+        self.m_skels = dict()
+        self.m_skelInfos = []
+    
+    def triangulate_person(self, skel2d):
```

Comment:
Names of skel2d and skel_info are confusing. For constant, we have a different name manner:
https://peps.python.org/pep-0008/#constants
Also, try not to use `skel` in the next review, which is not a common name shared by all contributors.

Reply:
`skel` means skeleton with imformation of keypoints and connections.
I will rename `skel` to `limb`.